### PR TITLE
Testing Your Ruby Code - Enumerable

### PR DESCRIPTION
**Pull request by guirecordon**
I wrote the test cases to all of my Enumerable methods and am now ready to request a review. 

I used the test cases provided in the Ruby docs documentation to write my own test cases. They were all separated in different it blocks as the project demands specified and I also included them inside of different context blocks for better organization and readability. 

Thank you.

### CODE REVIEW

**Review by kobiyoyo — CHANGES_REQUESTED** [review]
## Hi @guirecordon         
#### Great Job so far, but there is one thing to do .
### General
- [x] Lets fix linter errors

### Make changes and submit another code review request.
*Happy coding* 
(Any questions Feel free to contact me on slack @kobiyoyo)

**Comment by guirecordon** [comment]
I fixed linter errors as requested. 

Thank you.

File: spec/script_spec.rb [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+  describe '#my_all?' do
+    let(:number) { [1, 2i, 3.14] }
+
+    context 'when block is given' do
+      it 'returns true if the block never returns false or nil' do
+        expect(words.my_all?(&block1)).to eql(true)
+      end
+      it 'returns true if the block never returns false or nil' do
+        expect(words.my_all?(&block2)).to eql(false)
+      end
```
Comment (shshamim63) [inline spec/script_spec.rb]: Here you mentioned that `returns true if the block never returns false or nil` But you are expecting to return false in this scenario. Don't you think that the message should have a co-relation between expectation and the statement? I would rather say `returns false if the block returns false or nil`.

**Review by shshamim63 — APPROVED** [review]
## Status :100: Approved
___
Hi @guirecordon :wave: 
Amazing job :clap: similar tests are grouped properly, easy to read the tests (you have added _ between numbers to make it more readable, loved it). Well some messages are not properly covering the aspect. I have attached an example in the comment below. And I am sure that you will brush them in your upcoming days. Nothing is blocking me to approve it right now. 
___
##### This :+1: is from me you can go ahead and merge it.
___
## :herb: Happy Coding :herb: 
### Shakhawat Hossain

File: spec/script_spec.rb (hunk 1 above)
Comment (guirecordon) [reply spec/script_spec.rb]: Yes, that makes sense. Thanks!

**Pull request merged**